huaweicloud/resource_huaweicloud_cce_addon_v3_test.go

```diff
@@ -194,16 +194,40 @@ func testAccCCEAddonV3_values(rName string) string {
 	return fmt.Sprintf(`
 %s
 
+resource "huaweicloud_cce_node_pool" "test" {
```

Comment:
is `huaweicloud_cce_node_pool` necessary for this testing case?

Reply:
yes, addon autoscaler should be used with node pool.
Scaling is not support in default node pool, so installing autoscaler without node pool is actually meanless.